### DISCUSSION TITLE
Can i use efi for Lenovo T440s i7-4600U

### DISCUSSION THREAD

**Discussion by alfredoptarigan** [MultimediaLucario/Lenovo-ThinkPad-T440s#23] [Q&A]
I want use this efi, but can it run on Intel i7-4600U?

**Comment by johnnybainguyen** [comment]
I also have the i7-4600u and would like to know if I could use this efi.  Thanks!

**Reply by MultimediaLucario** [reply]
I mean you could try but I have heard conflicting results.

**Comment by BritoHTK** [comment]
I also have an i7 4600u, did you find any solution?
